bitbots_vision/src/bitbots_vision/vision_modules/yolo_handler.py

```diff
@@ -7,7 +7,7 @@
 except ImportError:
     rospy.logerr("Not able to run Darknet YOLO! Its only executable under python3 with yolo34py or yolo34py-gpu installed.", logger_name="vision_yolo")
 import numpy as np
-from .candidate import CandidateFinder, BallDetector, Candidate
+from .candidate import CandidateFinder, CandidateFinder, Candidate
```

Comment:
```suggestion
from .candidate import CandidateFinder, Candidate
```
As suggested by code factor, please resolve this.